charts/fluentd/templates/secret.yaml

```diff
@@ -1,9 +1,11 @@
+{{- if not .Values.secretName -}}
 apiVersion: {{ .Values.apiVersions.secret }}
 kind: Secret
 metadata:
-  name: {{ .Values.secretName }}
+  name: logzio-logs-secret
```

Comment:
Is there a reason this is hard coded and not using `{{ .Values.defaultSecretName }}`?

Reply:
@mirii1994 I missed this one. Looking at the code again, I have made some changes which follow a more common helm values pattern